docs/index.rst

```diff
@@ -3,9 +3,9 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-=================================
-Welcome to oemof's documentation!
-=================================
+=======================================
+Welcome to oemof solph's documentation!
```

Comment:
Here it should be oemof-solph, right? So the hyphen needs to be added.
In general, I like your idea of making it clear in the docs that this is oemof-solph, not oemof.